### DISCUSSION TITLE
Automatic updates on LXC containers

### DISCUSSION THREAD

**Discussion by DJKatastrof** [tteck/Proxmox#1601] [General]
Hi!

I want to start by saying, thank you so much for these scripts. 
Helped me a lot.

Is there any way to check for updates and use your scripts to update containers?

Thank you.

**Comment by tteck** [comment]
Have you tried the "Proxmox VE LXC Updater" script?

**Comment by DJKatastrof** [comment]
No way, I missed that...
Thank you!
